Sources/ComposableArchitecture/Instrumentation.swift

```diff
@@ -57,11 +59,16 @@ public class Instrumentation {
   public typealias Callback = (_ info: CallbackInfo<Any, Any>, _ timing: CallbackTiming, _ kind: CallbackKind) -> Void
   let callback: Callback?
 
+  /// Used to track when an instance of a `ViewStore` was created
+  public typealias ViewStoreCreatedCallback = (_ instance: AnyObject, _ file: StaticString, _ line: UInt) -> Void
```

Comment:
Wouldn't it be more elegant to just extend CallbackKind above by adding `.viewStoreCreated` and use the existing callback mechanism?
That would also allow to keep the class responsibility more to the point and avoid this `In addition to X it also does Y` in the comment of the class.

Reply:
It doesn't really fit since it's not for timing things, I used Instrumentation because it's accessible Singleton we have and it's already connected to all interested parties, it's either that or another global variable just for the ViewStore callback (which I had in my local fork)